config/ROOT.scons

```diff
@@ -28,7 +28,7 @@ rootdict = Builder(generator = generate_action,
 rootenv = Environment(ENV={'PATH' : os.environ['PATH']})
 rootenv.ParseConfig("root-config --cflags --ldflags --libs")
 rootenv.Append(LIBS='-lSpectrum')
-rootenv.Append(LIBS=['-lFoam', '-lPyROOT'])
+rootenv.Append(LIBS=['-lFoam', '-lPyROOT','-lMinuit2'])
```

Comment:
Yes, definitely -- it would be very helpful if the build process would notice lMinuit2 missing and fail immediately with a friendly error message.

(On the other hand I'd caution against detecting the library and automatically enabling/disabling code that uses it, since this will just lead to confusion and frustration later).